README.md

```diff
@@ -39,14 +39,51 @@ A `validate.sh` script is provided for convenience.
 
 Note: JSON Schemas only capture the NGSI simplified representation (`options=keyValues`)
 
+
+
+
 ## How to contribute
 
-Contributions should come in the form of pull requests. 
+Contributions should come in the form of pull requests.
+
+New data models should include:
+- A data model description based on the [data model template](https://github.com/Fiware/dataModels/blob/master/datamodel_template.md), e.g. [spec.md](https://github.com/Fiware/dataModels/blob/master/Weather/WeatherObserved/doc/spec.md) (model specifications should be stored in the doc folder of the data model)
+- A summary file (as an extract from the spec file), e.g. [README.md](https://github.com/Fiware/dataModels/blob/master/Weather/WeatherObserved/README.md)
+- A JSON Schema definition, e.g. [schama.json](https://github.com/Fiware/dataModels/blob/master/Weather/WeatherObserved/schema.json)
+- A JSON example file, e.g.  [example.json](https://github.com/Fiware/dataModels/blob/master/Weather/WeatherObserved/example.json)
+
+### Continuous integration
+To facilitate the validation of changes on existing models, and ensuring the validity of new ones, we introduced continuous integration tests. Pull requests will require the passing of continuous integration tests.
+
+Contributors of new data models, should
+- Create a new set of tests for their models and store in it in the ```test``` folder as shell script;
+- Include the set of tests in the continuous integration workflow in the ```.travis.yml``` file.
+
+Example of integration test:
```

Comment:
I'd not put here the example, to avoid duplication and risk of becoming outdated. A link to the actual file with that test should suffice.